frontend/package.json

```diff
@@ -15,7 +15,27 @@
     "lint:hbs": "ember-template-lint .",
     "lint:js": "eslint .",
     "start": "ember serve",
-    "test": "ember test"
+    "test": "ember test",
+    "on:precommit": "lint-staged"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "yarn on:precommit",
+      "post-commit": "git update-index --again"
+    }
+  },
+  "lint-staged": {
+    "./*/{app,addon,tests,config}/**/*.{js,ts,json,css}": [
+      "node ./node_modules/prettier/bin-prettier.js --write",
+      "git add"
+    ],
+    "./*/{app,addon,tests}/**{!dummy}/*.hbs": [
```

Comment:
1) shouldn't it start from repo root? Like so: `./frontend//{app,addon,tests,config}/**/*.{js,ts,json,css}"`?
2) do we need `{addon}` and `{!dummy}`?